Jenkinsfile

```diff
@@ -1052,15 +1053,35 @@ pipeline {
                         label 'ci_vm1'
                     }
                     steps {
-                        unitTest timeout_time: 60,
+                        unitTest timeout_time: 30,
+                                 inst_repos: pr_repos(),
+                                 inst_rpms: unit_packages()
+                    }
+                    post {
+                      always {
+                            unitTestPost artifacts: ['unit_test_logs/*'],
```

Comment:
```suggestion
                          unitTestPost artifacts: ['unit_test_logs/*'],
```

Reply:
I don't quite get this change, but there were some whitespace issues on the following line which I've resolved.